lib/message.rb

```diff
@@ -0,0 +1,47 @@
+class Message
+
+  def welcome_prompt
+    "I have generated a beginner sequence with four elements made up of: (r)ed,
+    (g)reen, (b)lue, and (y)ellow. Use (q)uit at any time to end the game.
+    What's your guess?"
+  end
+
+  def turn_result
+    "'RRGB' has 3 of the correct elements with 2 in the correct positions
```

Comment:
you can edit this line so that it actually takes in the results from the `pins` that the `player` places